modules/ROOT/pages/configuration/plugins.adoc

```diff
@@ -2,71 +2,73 @@
-The JAR files for these products are located in the _products_ folder and can be installed as plugins.
+== Overview
 
-Neo4j Community Edition comes with the APOC plugin only in the _labs_ directory.
+Plugins are Java Archive (_.jar_) files that extend the functionality of Neo4j by adding new features and capabilities, such as graph algorithms, data integration, visualization, and monitoring tools.
 
-If you want to use your own plugins, ensure that you add them to the designated _plugins_ directory.
-This directory serves as the central location where Neo4j looks for and loads the plugins at startup.
+Both Neo4j Community Edition (CE) and Enterprise Edition (EE) come with a range of pre-installed products, such as Gen AI, Graph Data Science, and Fleet management in the _products_ directory and the APOC Core jar file in the _labs_ directory. +
+The Fleet management jar file is bundled with the Neo4j server starting with Neo4j 5.26.10 and 2025.07.0 onwards.
+If you are using an earlier version of Neo4j, you can get the matching version of the Fleet management plugin by following the download link that matches your server version in the *Monitor Deployment* wizard in the Aura console.
+See link:{neo4j-docs-base-uri}/aura/fleet-management/setup/[Aura documentation -> Fleet Management -> Add a deployment] for more information.
 
 [NOTE]
 ====
-Bloom Enterprise and GDS Enterprise require a license activation key.
-Reach out to your Neo4j account representative or request a representative to link:https://neo4j.com/contact-us/#sales-inquiry[contact you].
+Using the Fleet management plugin in Neo4j CE deployments is subject to some limitations due to the limited set of features, such as no metrics being available.
+Also, it is offered on a best-effort basis.
+Neo4j does not offer any technical support for it.
 ====
 
+Some of these plugins, such as Bloom and GDS Enterprise require a license activation key.
+Reach out to your Neo4j account representative or request a representative to link:https://neo4j.com/contact-us/#sales-inquiry[contact you].
+
+If you want to use your own plugins or any of the other supported plugins, such as Apoc Extended and Neo4j Semantics, ensure that you download and add them to the _plugins_ directory.
+See their respective documentation for more information.
+
+[[supported-plugins]]
+== Supported plugins and documentation
+
 The following plugins are supported:
 
-.Supported Neo4j plugins
+.Supported Neo4j plugins and documentation
 [options="header",cols="d,m,b,a"]
 |===
-|Name |Key  | License activation key required | Further information
+|Name |Key  | License activation key required | Documentation
+
+| APOC Core
+| `apoc`
+|
+| link:https://neo4j.com/docs/apoc/current/[APOC Core documentation]
 
 | APOC Extended
 | `apoc-extended`
 |
 | link:https://neo4j.com/labs/apoc/5/[APOC Extended user guide (Labs project)]
```

Comment:
There is also APOC Extended for 2025 -- https://neo4j.com/labs/apoc/2025/overview/. Should we link to them?

Reply:
Ah, right!